recipes/dlib/lapack_sgetrf.patch

```diff
@@ -0,0 +1,20 @@
+--- dlib/cmake_find_blas.txt
++++ dlib/cmake_find_blas.txt
+@@ -140,15 +140,8 @@
+          set(blas_found 1)
+          message(STATUS "Found OpenBLAS library")
+          set(CMAKE_REQUIRED_LIBRARIES ${blas_libraries})
+-         # If you compiled OpenBLAS with LAPACK in it then it should have the
```

Comment:
This just didn't seem to work? But I'm not sure if it was a quirk of our OpenBLAS build or not. The function `sgetrf_single` is in the binary, but it *is* marked as private - so this fails. I don't think that OpenBLAS is being used a lot with dlib. Again, it would be up to @davisking if he wanted to remove this check in general.